packages/react-client/src/index.ts

```diff
@@ -1,39 +1,28 @@
-export { FishjamProvider } from "./FishjamProvider";
-export { useCamera } from "./hooks/devices/useCamera";
-export { useInitializeDevices } from "./hooks/devices/useInitializeDevices";
-export { useMicrophone } from "./hooks/devices/useMicrophone";
-export type { JoinRoomConfig } from "./hooks/useConnection";
-export { useConnection } from "./hooks/useConnection";
-export type { PeerWithTracks } from "./hooks/usePeers";
-export { usePeers } from "./hooks/usePeers";
-export { useScreenShare } from "./hooks/useScreenShare";
+export { FishjamProvider, type FishjamProviderProps } from "./FishjamProvider";
+export { useCamera, type UseCameraResult } from "./hooks/devices/useCamera";
+export { useInitializeDevices, type UseInitializeDevicesResult } from "./hooks/devices/useInitializeDevices";
+export { useMicrophone, type UseMicrophoneResult } from "./hooks/devices/useMicrophone";
+export { type JoinRoomConfig, useConnection, type UseConnectionResult } from "./hooks/useConnection";
+export { type PeerWithTracks, usePeers, type UsePeersResult } from "./hooks/usePeers";
+export { useScreenShare, type UseScreenshareResult } from "./hooks/useScreenShare";
```

Comment:
I have mixed feelings if such change make it more readable 🤔 (I think I prefer old version).

But if we want to introduce such changes, it would be good to have eslint rule that forces one or another way. Just to avoid people changing it back and forth.